nixos/modules/hardware/bitcraze.nix

```diff
@@ -0,0 +1,23 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.hardware.bitcraze;
+
+in
+{
```

Comment:
This module feels a bit small. That isn't a problem in and of itself, but is there nothing else useful it could or should do? Like, would it make sense to add `cfclient` to `environment.packages` on `enable` for example?

Reply:
Not sure. This module pretty much only has to set up udev and the plugdev group, which is required if you want to use the hardware via e.g. the python library as well. I don't think the module needs to impose which software should be installed in addition.

Reply:
Right, that makes sense. Feel feel free to mark as resolved.